src/analyze/analyze-security.c

```diff
@@ -143,12 +147,11 @@ static bool security_info_runs_privileged(const struct security_info *i)  {
         return isempty(i->user);
 }
 
-static int assess_bool(
-                const struct security_assessor *a,
-                const struct security_info *info,
-                const void *data,
-                uint64_t *ret_badness,
-                char **ret_description) {
+static int assess_bool(const struct security_assessor *a,
+                       const struct security_info *info,
+                       const void *data,
+                       uint64_t *ret_badness,
+                       char **ret_description) {
```

Comment:
The style with arguments on a separate line has two advantages:
- indentation doesn't change if the function name changes length, reducing churn
- when multiple functions are defined (in .h files) all arguments have the same indentation so it looks better

This style of indentatino is pretty widely used in systemd. Please exclude this change.

Reply:
part of clang-format, will investigate